TwitchRSS/twitchrss.py

```diff
@@ -128,14 +130,25 @@ def fetch_user(channel_name):
 
 @cached(cache=TTLCache(maxsize=1000, ttl=VODCACHE_LIFETIME))
 def fetch_vods(channel_id):
-    return fetch_json(channel_id, VOD_URL_TEMPLATE)
+    clipFilter = request.args.get('filter')
+    
+    if clipFilter not in VALID_URL_ARGS:
+        clipFilter = 'all'
+    if clipFilter is None:
+        clipFilter = 'all'
+
+    return fetch_json(channel_id, VOD_URL_TEMPLATE, clipFilter)
 
 
-def fetch_json(id, url_template):
+def fetch_json(id, url_template, clipFilter = None):
     #update the oauth token
     token = authorize()
+    if clipFilter is not None:
```

Comment:
Nit: you can just put `if clipFilter:` here since `clipFilter` will not be empty string anyway which would be also falsey.